q2_diversity/_alpha/_visualizer.py

```diff
@@ -48,12 +48,20 @@ def alpha_group_significance(output_dir: str, alpha_diversity: pd.Series,
     filtered_columns = pre_filtered_cols - set(metadata.columns)
 
     if len(metadata.columns) == 0:
+        sample_ids = ', '.join(alpha_diversity.index)
         raise ValueError(
-            "Metadata does not contain any columns that satisfy this "
-            "visualizer's requirements. There must be at least one metadata "
-            "column that contains categorical data, isn't empty, doesn't "
-            "consist of unique values, and doesn't consist of exactly one "
-            "value.")
+            "Either the metadata file does not meet the requirements of this "
```

Comment:
Generally, this lgtm! I'll pull this down locally to test and make sure nothing looks weird on the command line, but otherwise this should be g2g 👍